src/viewport.js

```diff
@@ -4,6 +4,12 @@
 */
 export function inViewport (element, options) {
 
+    /**
+    * Check if window is undefined. Will not break
+    * isomorphic applications
+    */
+    if (typeof window === 'undefined') return
```

Comment:
I'm not sure we need to check here as well, because this won't ever be fired if none of the event handlers are registered and the API isn't defined.